modules/web-console/frontend/app/app.ts

```diff
@@ -175,8 +181,9 @@ export default angular
         'ui.grid.resizeColumns',
         'ui.grid.saveState',
         'ui.grid.selection',
-        'ui.router',
-        'ui.router.state.events',
+        uiRouter,
+        upgradeModule.name,
+        // 'ui.router.state.events',
```

Comment:
Why  "ui.router.state.events" - commented?

Reply:
Removed, it's no longer used in master either.